### DISCUSSION TITLE
Cannot access shadow element with seleniumbase

### DISCUSSION THREAD

**Discussion by hiyamandal** [seleniumbase/SeleniumBase#1869] [Q&A] [answered]
<img width="538" alt="image" src="https://user-images.githubusercontent.com/15275310/236301338-8cf8b227-3947-436b-a264-25826061cd0a.png">
Hello, 

I am trying the access the button.pagination__next-prompt (highlighted) which is a shadow element. I am using the code:
self.click('scale-pagination.hydrated::shadow div.pagination div.pagination__button-wrapper button.pagination__next-prompt')

but, it cannot access the element. The error is thrown as:
selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element is not clickable at point (710, 1034)

What am I doing wrong?

**Comment by mdmintz** [comment, marked as answer]
Usually with ``ElementClickInterceptedException: Message: element click intercepted: Element is not clickable at point`` there is another element that would receive the click (one that is likely hovering over the element you want in the UI). You need to either remove or hide that other element that is covering over the element you want. It's probably an overlay of some kind.

But otherwise, shadow elements are accessed via the ``::shadow`` selector, as you have done, with more examples here:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/shadow_root_test.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_shadow_dom.py

**Reply by hiyamandal** [reply]
I just checked the issue is only for the Chrome browser.. I tried it on Safari, and it works, but for Chrome if I use the API is_element_visible or is_element_clickable, the return is True, but the click API is throwing the error. 
On the other hand, it works perfectly with Safari!
